com.github.cubitect.cubiomes-viewer.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <id>com.github.cubitect.cubiomes-viewer</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+
+  <developer_name>Cubitect</developer_name>
+  <name>Cubiomes Viewer</name>
+  <summary>An efficient graphical Minecraft seed finder and map viewer.</summary>
+  <description>
+    <p>Cubiomes Viewer offers highly customizable seed-finding utilities and a map viewer for the Minecraft biome and structure generation.</p>
+  </description>
+
+  <releases>
+    <release version="2.3.1" date="2022-08-08">
+    </release>
+  </releases>
+
+  <launchable type="desktop-id">com.github.cubitect.cubiomes-viewer.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/cubitect/cubiomes-viewer/blob/trunk/etc/screenshot_maingui.png</image>
```

Comment:
this link loads HTML content. It's not the right URL for the PNG data.
Also it should be linking to a tag or commit, otherwise it will fail whenever the file is removed in master.

Reply:
Thanks for all the help.
Can I reference a path relative to the source as well, given that the screenshot is part of the tar ball?

Reply:
no, you can't.